changelog/bugfixes/2022-05-22-bootengine.md

```diff
@@ -0,0 +1 @@
+- fixed OEM ID from `pxe` to `metal` to follow the Ignition upgrade ([coreos-overlay#1874](https://github.com/flatcar-linux/coreos-overlay/pull/1874))
```

Comment:
```suggestion
- Fixed OEM ID from `pxe` to `metal` to follow the Ignition upgrade ([coreos-overlay#1874](https://github.com/flatcar-linux/coreos-overlay/pull/1874))
```

Reply:
Also, it would be better to include the bootengine PR instead of the coreos-overlay